packages/refdata-extract-processor/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@defra/wls-refdata-extract-processor",
+  "version": "1.2.2",
+  "description": "Interface to Microsoft Dynamics. Extract processor for reference data.",
+  "main": "",
+  "type": "module",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=16.13.0"
+  },
+  "scripts": {
+    "start": "node src/processor.js",
+    "test": "echo \"Error: run tests from root\" && exit 1"
```

Comment:
Bonus brownie points if you can modify the 'test' command or remove it, but that's just for tidiness. No biggie.

Reply:
Sounds like a last thing Friday job! I will bear it in mind.